x-pack/plugins/spaces/public/space_selector/space_selector.tsx

```diff
@@ -106,104 +104,111 @@ export class SpaceSelector extends Component<Props, State> {
-                  defaultMessage="You can change your space at anytime"
-                />
-              </p>
-            </EuiText>
-          </EuiPageHeader>
-          <EuiPageContent className="spcSpaceSelector__pageContent">
-            <EuiFlexGroup
-              // @ts-ignore
-              direction="column"
-              alignItems="center"
-              responsive={false}
-            >
-              {this.getSearchField()}
-            </EuiFlexGroup>
-
-            <EuiSpacer size="xl" />
-
-            {this.state.loading && <EuiLoadingSpinner size="xl" />}
-
-            {!this.state.loading && (
-              <SpaceCards spaces={filteredSpaces} serverBasePath={this.props.serverBasePath} />
-            )}
-
-            {!this.state.loading && !this.state.error && filteredSpaces.length === 0 && (
-              <Fragment>
-                <EuiSpacer />
-                <EuiText color="subdued" textAlign="center">
+      <KibanaPageTemplate
+        template="empty"
+        className="spcSpaceSelector"
+        data-test-subj="kibanaSpaceSelector"
+        pageContentBodyProps={{ className: 'spcSpaceSelector__pageContent' }}
+      >
+        <EuiText textAlign="center" size="s">
+          <EuiSpacer size="xxl" />
+          <KibanaSolutionAvatar name="Elastic" size="xl" />
+          <EuiSpacer size="xxl" />
+          <h1
+            // plain `eui` class undos forced focus style on non-EUI components
+            className="eui spcSpaceSelector__pageHeader"
+            tabIndex={0}
+            ref={this.setHeaderRef}
+          >
+            <FormattedMessage
+              id="xpack.spaces.spaceSelector.selectSpacesTitle"
+              defaultMessage="Select your space"
+            />
+          </h1>
+          <EuiTextColor color="subdued">
+            <p>
+              <FormattedMessage
+                id="xpack.spaces.spaceSelector.changeSpaceAnytimeAvailabilityText"
+                defaultMessage="You can change your space at anytime."
+              />
+            </p>
+          </EuiTextColor>
+        </EuiText>
+        <EuiSpacer size="xxl" />
+
+        {this.getSearchField()}
+
+        <EuiSpacer size="xl" />
+
+        {this.state.loading && <EuiLoadingSpinner size="xl" />}
+
+        {!this.state.loading && (
+          <SpaceCards spaces={filteredSpaces} serverBasePath={this.props.serverBasePath} />
+        )}
+
+        {!this.state.loading && !this.state.error && filteredSpaces.length === 0 && (
+          <Fragment>
+            <EuiSpacer />
+            <EuiPanel className="spcSpaceSelector__errorPanel" color="subdued">
+              <EuiTitle size="xs">
+                <h2>
+                  {i18n.translate(
+                    'xpack.spaces.spaceSelector.noSpacesMatchSearchCriteriaDescription',
+                    {
+                      defaultMessage: 'No spaces match "{searchTerm}"',
+                      values: { searchTerm: this.state.searchTerm },
```

Comment:
(untested) I wonder if it's getting confused with the nested quotes within `defaultMessage`. What if we move the quotes into the `values` object? I see this pattern being done elsewhere:

https://github.com/elastic/kibana/blob/c048f71626af92251def42a3996f2c382f36cc46/packages/kbn-es-query/src/kuery/kuery_syntax_error.ts#L75-L81

```suggestion
                      defaultMessage: 'No spaces match {searchTerm}',
                      values: { searchTerm: `"${this.state.searchTerm}"` },
```

Reply:
Trying it out